sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/Custom/Models/ContainerRegistryImportSource.Serialization.cs

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// <auto-generated/>
-
 #nullable disable
 
 using System.Text.Json;
 using Azure.Core;
 
```

Comment:
I don't prefer to totally remove this model to the customized code, it might let us miss some new properties if the service team update this model in the future. How about we rename the property to `RegistryAddress` and write some code as shown below in the customized code:
```csharp
[Obsolete("RegistryUri is deprecated, use RegistryAddress instead")]
[EditorBrowsable(EditorBrowsableState.Never)]
public Uri RegistryUri {
    get
    {
        if (RegistryAddress != null && Uri.TryCreate(RegistryAddress, UriKind.RelativeOrAbsolute, out var registryUri))
            return registryUri;
        return null;
    }
    set
    {
        RegistryAddress = value == null ? null : value.ToString();
    }
}
```

Reply:
This seems a good workaround to avoid customizing the `write` method in serialization class (and the generator does not support only customize `write` method, that's why I need to customize the whole class).

Reply:
Relative Uri is discouraged with https://github.com/Azure/azure-sdk-for-net/blob/main/eng/BannedSymbols.txt, will modify to only use AbsoluteUri.